yarn.lock

```diff
@@ -9446,7 +9446,7 @@ make-fetch-happen@^10.0.3, make-fetch-happen@^10.0.6:
   dependencies:
     agentkeepalive "^4.2.1"
     cacache "^16.1.0"
-    http-cache-semantics "^4.1.1"
+    http-cache-semantics "^4.1.0"
```

Comment:
It seems odd that this version would descend, but it's already set to 4.1.1 in the yarn.lock file. so we're not impacted by https://www.cve.org/CVERecord?id=CVE-2022-25881